packages/notifi-axios-adapter/lib/fragments/supportedTargetTypesFragment.ts

```diff
@@ -0,0 +1,7 @@
+export const supportedTargetTypesFragment = `
+fragment supportedTargetTypesFragment on SupportedTargetTypesForDapp {
+  
```

Comment:
intentionally fetching nothing for now?

Reply:
nope, definitely not my intention!

Reply:
query supportedTargets {
  supportedTargetTypesForDapp(getSupportedTargetTypesForDappInput: {
    dappAddress: ""
  }) {
    
  }